process/message_no_zstd.go

```diff
@@ -0,0 +1,476 @@
+//go:build !cgo
```

Comment:
This allows us to import `process` pkg when cgo is disabled.
This file is copy-pasted form `process/message_zstd.go`.
I only changed two functions

https://github.com/DataDog/agent-payload/blob/ed71f54d28014e3fe3fabea2e85b088d7d99a1c1/process/message_no_zstd.go#L56-L58


https://github.com/DataDog/agent-payload/blob/ed71f54d28014e3fe3fabea2e85b088d7d99a1c1/process/message_no_zstd.go#L337-L339